Utils/Dataflow/test/pyDKB/test.sh

```diff
@@ -47,22 +49,27 @@ test_case() {
   case=$1
   case_id=`basename $case`
 
+  base_dir_mod=`echo $base_dir | sed -e "s|test/pyDKB||g"`
+
   before=`cat $case/before 2>/dev/null`
   [ -n "$before" ] && before="$before;"
   cmd=`cat $case/cmd`
   after=`cat $case/after 2>/dev/null`
-
   eval "$before $cmd; $after"  2>&1 1> out.tmp | \
-    grep -a -v '(WARN) (pyDKB.dataflow.cds) Submodule failed (No module named invenio_client.contrib)' | \
+    grep -a -v "(WARN) (pyDKB.dataflow.cds) Submodule failed (No module named 'invenio_client')" | \
     sed -E -e"s#$base_dir#\$base_dir#" \
+           -e"s#$base_dir_mod##" \
+           -e's#\$base_dir/./../../##g' \
            -e"s#^[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2} ##" \
-           -e"s#, line [0-9]+#, line <NNN>#" > err.tmp
+           -e"s#, line [0-9]+#, line <NNN>#" \
+           -e"s#/.*/os.py#os.py#" | sort > err.tmp
```

Comment:
Why the unsorted output no longer works?

Reply:
As soon as there are changes in exception handling and print order is changed it was decided to use already prepared tests in python2 for python3. By using this technique  we avoided writing new tests for a python3 code.